extras/create_send_notifications.py

```diff
@@ -42,26 +42,41 @@ def create_output_directories():
         os.path.join(NOTIFICATIONS_BASE_DIR, NOTIFICATION_ARCHIVE_DIR)
     )
 
+def build_notifications_org_list(db):
+    """Return notifications organization list.
 
-def build_cyhy_org_list(db):
-    """Build list of CyHy organization IDs.
-
-    This is the list of CyHy organization IDs (and their descendants) that
-    receive CyHy reports.
+    This is the list of organization IDs that should
+    have a notification generated and sent.
+    """
+    notifications_to_generate = set()
+    notifications_to_delete = set(ticket_owner_ids) - notifications_to_generate
+    cyhy_parent_ids = set()
+    ticket_owner_ids = db.notifications.distinct("ticket_owner")
```

Comment:
This line needs to be moved to the end of this function, just before the `return` statement.  It will throw an error if you don't move this, since `ticket_owner_ids` is undefined at this point.